UnitTest_Engine/Compute/MergeTestDataSets.cs

```diff
@@ -0,0 +1,127 @@
+﻿/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using BH.oM.Base;
+using BH.oM.Base.Attributes;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using BH.oM.Data.Library;
+using UT = BH.oM.Test.UnitTests;
+
+namespace BH.Engine.UnitTest
+{
+    public static partial class Compute
+    {
+        /***************************************************/
+        /**** Public Methods                            ****/
+        /***************************************************/
+
+        [Description("Merges two Dataset of UnitTests into one Dataset.")]
+        [Input("set1", "First Dataset to merge.")]
+        [Input("set2", "Second Dataset to merge.")]
+        [Output("mergedDataset", "Dataset containing UnitTests and source information from both provided Datasets.")]
+        public static Dataset MergeTestDataSets(Dataset set1, Dataset set2)
```

Comment:
Would it make any sense to accept a list of Dataset as an input and merge all datasets in the list into one?

Reply:
Can add a method like that as well, just chain calling this method if you like. Still think it is useful to have this one accepting two specific ones as well.

Reply:
Yeah that's what I'm thinking

Reply:
Done :) https://github.com/BHoM/Test_Toolkit/pull/404/commits/18bc4119b0e63e9f51e82ec0513b144e63a034c2